config/manager/kustomization.yaml

```diff
@@ -16,3 +16,15 @@ images:
   newTag: 4.14.0
 patches:
 - path: related-images/patch.yaml
+- patch: |
```

Comment:
why not just update the deployment spec directly, https://github.com/openshift-kni/lifecycle-agent/blob/main/config/manager/manager.yaml#L13

Reply:
Updated to include the annotation directly